google/cloud/vision/color.py

```diff
@@ -0,0 +1,191 @@
+
+        :type response: dict
+        :param response: Dictionary response from Vision API with image
+                         properties data.
+
+        :rtype: :class:`~google.cloud.vision.color.ImagePropertiesAnnotation`.
+        :returns: Populated instance of ``ImagePropertiesAnnotation``.
+        """
+        colors = [ColorInformation.from_api_repr(color) for color in
+                  response['dominantColors']['colors']]
+        return cls(colors)
+
+    @property
+    def colors(self):
+        """Colors in an image.
+
+        :rtype: list of :class:`~google.cloud.vision.color.ColorInformation`
+        :returns: Populated list of ``ColorInformation``.
+        """
+        return self._colors
+
+
+class Color(object):
+    """Representation of RGBA color information.
+
+    :type red: int
+    :param red: The amount of red in the color as a value in the interval
+                [0, 1].
+
+    :type green: int
+    :param green: The amount of green in the color as a value in the interval
+                  [0, 1].
+
+    :type blue: int
+    :param blue: The amount of blue in the color as a value in the interval
+                 [0, 1].
+
+    :type alpha: float
+    :param alpha: The fraction of this color that should be applied to the
+                  pixel.
+    """
+    def __init__(self, red, green, blue, alpha):
+        self._red = red
+        self._green = green
+        self._blue = blue
+        self._alpha = alpha
+
+    @classmethod
+    def from_api_repr(cls, response):
+        """Factory: construct a ``Color`` from a Vision API response.
+
+        :type response: dict
+        :param response: Color from API Response.
+
+        :rtype: :class:`~google.cloud.vision.color.Color`
+        :returns: Instance of :class:`~google.cloud.vision.color.Color`.
+        """
+        red = response['red']
+        green = response['green']
+        blue = response['blue']
+        alpha = response.get('alpha')
+
+        return cls(red, green, blue, alpha)
+
+    @property
+    def red(self):
+        """Red color.
+
+        :rtype: int
+        :returns: Red RGB value.
+        """
+        return self._red
+
+    @property
+    def green(self):
+        """Green color.
+
+        :rtype: int
+        :returns: Green RGB value.
+        """
+        return self._green
+
+    @property
+    def blue(self):
+        """Blue color.
+
+        :rtype: int
+        :returns: Blue RGB value.
+        """
+        return self._blue
+
+    @property
+    def alpha(self):
+        """Alpha level.
+
+        :rtype: float
+        :returns: Alpha transparency level.
+        """
+        return self._alpha
+
+
+class ColorInformation(object):
+    """Representation of color information from API response.
+
+    :type color: :class:`~google.cloud.vision.color.Color`
+    :param color: RGB components of the color.
+
+    :type score: float
+    :param score: Image-specific score for this color. Value in range [0, 1].
```

Comment:
So I _think_ it represents how dominant the color is in the image. I can't seem to find documentation to back that up though.